src/components/views/game/game/yourhand/YourHand.tsx

```diff
@@ -1,35 +1,34 @@
-import React, { useState } from "react"
+import React from "react"
 import { useCards } from "../../../../../hooks/api/useCards"
 import { useCurrentGame } from "../../../../../hooks/api/useCurrentGame"
 import { CardComponent } from "../../../../ui/CardComponent"
 import "./YourHand.scss"
+import { toJS } from "mobx" //DEBUGGING - REMOVE LATER
+import { observer } from "mobx-react-lite"
 
-export const YourHand = () => {
+export const YourHand = observer(() => {
   const { updatecards } = useCards()
-  const { activeMatch, yourActiveHand } = useCurrentGame()
-  const [opacity, setOpacity] = useState(10)
+  const { activeMatch, yourActiveHand, activeRound } = useCurrentGame()
 
   const notYetPlayed =
     yourActiveHand?.cards.filter((value) => value.round === null) ?? []
 
   const clickCard = async (card, number) => {
-    setOpacity(number)
-    updatecards(card)
+    setTimeout(() => updatecards(card), 600)
+    console.log(toJS(activeRound.cards)) //DEBUGGING - REMOVE LATER
   }
 
   return (
     <div className="your-hand">
-      <p>These are the Cards for Match {activeMatch?.matchNumber}:</p>
+      <p style={{ marginLeft: "10px" }}>
+        These are the Cards for Match {activeMatch?.matchNumber} Round{" "}
```

Comment:
```suggestion
        These are the Cards for Match {activeMatch?.matchNumber} Round 
```
Geht das nicht auch mit einem normalen space?